string sentence
---
bool success
int32 x
int32 y